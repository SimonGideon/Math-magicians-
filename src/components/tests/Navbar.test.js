import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import NavbarContainer from '../Navbar';

it('Rendered as expected', () => {
  const tree = renderer
    .create(
      <MemoryRouter>
        <NavbarContainer />
      </MemoryRouter>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
