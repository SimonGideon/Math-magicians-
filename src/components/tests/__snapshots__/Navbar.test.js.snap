// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendered as expected 1`] = `
<nav
  className="navbar"
>
  <a
    className="header"
    href="/"
    onClick={[Function]}
  >
    Math Magician
  </a>
  <ul
    className="pages"
  >
    <li>
      <a
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li>
      <a
        className="bordered"
        href="/Calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
    </li>
    <li>
      <a
        href="/Quotes"
        onClick={[Function]}
      >
        Quote
      </a>
    </li>
  </ul>
</nav>
`;
